app/controllers/providers/allocations_controller.rb

```diff
@@ -29,25 +29,19 @@ def repeat_request; end
 
     def create
       # TODO: we need to add error handling here
-      AllocationServices::Create.call(
+
+      allocation = AllocationServices::Create.call(
         accredited_body_code: @provider.provider_code,
         provider_id: @training_provider.id,
-        requested: requested?,
+        request_type: params[:request_type],
         number_of_places: params[:number_of_places],
```

Comment:
Where is `params[:number_of_places]` coming from? I don't see it in the form? Still working on it or am I missing something?

Reply:
this endpoint is use by multiple forms to create the allocation. in repeat request we don't send this as we will use last years allocation. for new and changing of requests this form field should be available and send to api